scripts/format_app.py

```diff
@@ -11,7 +11,7 @@
 
 def handle_cli_args():
   parser = argparse.ArgumentParser(description='A script for formatting a given app within emqx')
-  parser.add_argument("-a", "--application", required=True, help="The relative path to the application which is to be formatted.")
+  parser.add_argument("-a", "--application", required=True, action="append", help="The relative path to the application which is to be formatted.")
```

Comment:
Should this be pulled out into its own commit since it has nothing to do with telemetry? I don't know why it would, but if for some reason the telemetry commit had to be rolled back you'd lose this too.

Reply:
Yes, there is a commit with only this change: 15dcf8aaa3db60a408cc67d1298f6c7f6f6ce1f0